src/app/panels/middle/xvb.rs

```diff
@@ -222,11 +231,11 @@ impl crate::disk::state::Xvb {
                         if self.mode ==  XvbMode::ManualDonationLevel {
                             ui.radio_value(&mut self.manual_donation_level, ManualDonationLevel::Donor, "Donor")
                             .on_hover_text(XVB_DONATION_LEVEL_DONOR_HELP);
-                            ui.radio_value(&mut self.manual_donation_level, ManualDonationLevel::DonorVIP, "DonorVIP")
+                            ui.radio_value(&mut self.manual_donation_level, ManualDonationLevel::DonorVIP, "Donor VIP")
                             .on_hover_text(XVB_DONATION_LEVEL_VIP_DONOR_HELP);
-                            ui.radio_value(&mut self.manual_donation_level, ManualDonationLevel::DonorWhale, "DonorWhale")
+                            ui.radio_value(&mut self.manual_donation_level, ManualDonationLevel::DonorWhale, "Donor Whale")
                             .on_hover_text(XVB_DONATION_LEVEL_WHALE_DONOR_HELP);
-                            ui.radio_value(&mut self.manual_donation_level, ManualDonationLevel::DonorMega, "DonorMega")
+                            ui.radio_value(&mut self.manual_donation_level, ManualDonationLevel::DonorMega, "Donor Mega")
```

Comment:
implement Display trait on ManualDonationLevel enum and calling directly self.manual_donation_level.to_string()

Reply:
I don't see how I would do that since instead I used a Display trait and for each option did the below
```
ManualDonationLevel::Donor.to_string()
```

Reply:
I missread, thought there was two different enums. It looks good with your last commit.

Reply:
that's what I meant :)